l10n_nl_base_ubl/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Base UBL - Netherlands",
+    "version": "13.0.1.0.0",
+    "category": "Dutch Localization",
```

Comment:
Please Change category so it shows up corretly in odoo appstore and backend applist.
"category": "Localization",
or
"category": "Localization\Netherlands",